pages/common/git-worktree.md

```diff
@@ -0,0 +1,15 @@
+# git worktree
+
+> Manage multiple working trees attached to the same repository.
+
+- Create a new folder with the specified branch checked out into it:
+
+`git worktree add {{path/to/folder}} {{branch}}`
```

Comment:
Is it better to have them grouped by type of command, or listed in order of most commonly used to least commonly used?